allure-generator/src/main/javascript/components/modal/ModalView.js

```diff
@@ -28,7 +28,7 @@ class ModalView extends View {
         e.stopPropagation();
     }
 
-    @on('click .modal__background, .modal__close')
+    @on('click .modal__close')
```

Comment:
as far as I know usually modals can be closed by click on background, isn't?

Reply:
yes will revert, occassionally I would click off the attachment and close it accidently, but its quite a rare scenario and original behaviour is probably better